_studio/mfx_lib/encode_hw/h265/src/mfx_h265_encode_hw.cpp

```diff
@@ -126,9 +126,20 @@ bool GetRecInfo(const MfxVideoParam& par, mfxFrameInfo& rec)
     }
     else if (CO3.TargetChromaFormatPlus1 == (1 + MFX_CHROMAFORMAT_YUV422) && CO3.TargetBitDepthLuma == 10)
     {
-        rec.FourCC = MFX_FOURCC_Y210;
-        rec.Width /= 2;
-        rec.Height *= 2;
+#if (MFX_VERSION >= MFX_VERSION_NEXT)
+        if (par.m_platform >= MFX_HW_TGL_LP)
```

Comment:
What's going on here? What does these fourcc mean and why/how they are mapped to chroma formats? Is there driver caps query to replace platforms checks? If not what is libva bug number to request?